admin/class-help-center.php

```diff
@@ -208,7 +208,8 @@ public static function get_translated_texts() {
 			'searchResultDetail.backButtonLabel' => __( 'Go back to the search results', 'wordpress-seo' ),
 			'searchResultDetail.iframeTitle'     => __( 'Knowledge base article', 'wordpress-seo' ),
 			'searchResult.noResultsText'         => __( 'No results found.', 'wordpress-seo' ),
-			'searchResult.foundResultsText'      => __( 'Number of results found: { resultsCount }', 'wordpress-seo' ),
+            /* translators: %d expands to the number of results found . */
+			'searchResult.foundResultsText'      => sprintf(__( 'Number of results found: %d', 'wordpress-seo' ), '{ resultsCount }' ),
```

Comment:
`%d` won't work here, because you are replacing it for a string. I would change this to `%s`

Reply:
Can you change `__` in `esc_html__`

Reply:
It might be better to put things on newline, to make it more readable.
```
sprintf(
   /* translators: %s expands to the number of results found . */
   esc_html__( 'Number of results found: %s', 'wordpress-seo' ), 
   '{ resultsCount }' 
),
```